src/Microsoft.Health.Dicom.Operations.Functions.UnitTests/DurableFunctionsCleanupTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+//using System;
+//using System.Collections.Generic;
+//using System.Text;
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Operations.Functions;
+using Microsoft.Health.Dicom.Operations.Functions.Indexing.Configuration;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Functions.UnitTests
+{
+    public class DurableFunctionsCleanupTests
+    {
+        private readonly ReindexErrorConfiguration _cleanupConfig = new ReindexErrorConfiguration();
+
+        [Fact]
+        public async Task CleanupCompletedDurableFunctions_GivenValidInput_ReturnsStringListOfOrchestrationsAsync()
+        {
+            Console.WriteLine("Hello");
+            Assert.Equal(0, 0);
+            var request = GenerateHttpRequest();
+
+            //ILogger logger = NullLoggerFactory.Instance.CreateLogger("test");
```

Comment:
My rule of thumb is: if you're asserting logic, then substitute. Otherwise use the null or dummy implementation for "pass-through"